v0/destinations/clevertap/transform.js

```diff
@@ -223,6 +243,11 @@ const responseBuilderSimple = (message, category, destination) => {
     // ----------------------------------------------------------------------
     else {
       eventPayload = constructPayload(message, MAPPING_CONFIG[category.name]);
+
+      // If timestamp is not in unix format
+      if (eventPayload.ts && !Number(eventPayload.ts)) {
+        eventPayload.ts = toUnixTimestamp(eventPayload.ts);
+      }
```

Comment:
Let's do one thing instead of making changes in multiple locations, let's do this change just before Line no 261 in eventPayload